lib/rules/block-scoped-var.js

```diff
@@ -173,11 +173,20 @@ module.exports = function(context) {
             declareByNodeType(param);
         });
 
-        declare(node.id ? [node.id.name] : []);
         declare(node.rest ? [node.rest.name] : []);
         declare(["arguments"]);
     }
 
+    function functionDeclarationHandler(node) {
```

Comment:
Please add JSDoc comments to these functions. They are so similar that it would be hard for most to understand the difference.